src/components/Editor/Project/Project.scss

```diff
@@ -79,7 +80,7 @@
     display: flex;
     box-sizing: border-box;
     flex: 5;
-    overflow-y: hidden;
+    overflow: hidden;
```

Comment:
📓 overflows hidden to fix collapsable state in mobile/tablet (run button was off the screen) and multiple scrollbars on HTML output when output panel is small than the website size